### DISCUSSION TITLE
Is there any plan to support a mechanism for disabling the debug logs which log run queries?

### DISCUSSION THREAD

**Discussion by tonyalaribe** [tchoutri/pg-entity#39] [Q&A]
In production, and in applications with security constraints, it can be preferable not to log all the queries which are executed. Especially not with all the values in them, but also because logging comes with a cost (Financial to log aggregators, but also performance). Is such functionality to disable the logs (eg in prod) on the roadmap?

**Comment by tchoutri** [comment]
Yes I plan to integrate a real logging solution, like log-base, so that logs can be disabled, or enabled *and* structured. :)
